src/coreclr/nativeaot/Runtime/loongarch64/GcProbe.S

```diff
@@ -139,9 +134,8 @@ NESTED_ENTRY RhpGcProbeHijack, _TEXT, NoHandler
     jirl  $r0, $ra, 0
 
 LOCAL_LABEL(WaitForGC):
-    lu12i.w  $t7, ((DEFAULT_FRAME_SAVE_FLAGS + PTFF_SAVE_R4 + PTFF_SAVE_R5) >> 12) & 0xfffff
-    ori  $t7, $t7, (DEFAULT_FRAME_SAVE_FLAGS + PTFF_SAVE_R4 + PTFF_SAVE_R5) & 0xfff
-    or  $t3, $t3, $t7
+    lu12i.w  $t3, ((DEFAULT_FRAME_SAVE_FLAGS + PTFF_SAVE_R4 + PTFF_SAVE_R5 + PTFF_THREAD_HIJACK) >> 12) & 0xfffff
+    ori  $t3, $t3, (DEFAULT_FRAME_SAVE_FLAGS + PTFF_SAVE_R4 + PTFF_SAVE_R5 + PTFF_THREAD_HIJACK) & 0xfff
```

Comment:
Should `PTFF_THREAD_HIJACK` be `PTFF_THREAD_HIJACK_HI`? It should be giving a compiler error because PTFF_THREAD_HIJACK is undefined.
Noticed in #110688 when I was resolving the merged conflicts.

Reply:
Right, this should be `PTFF_THREAD_HIJACK_HI` or `PTFF_THREAD_HIJACK` needs to be defined. Either way should work. It would be a trivial fix.

I'd be more worried whether this runs correctly. it should, but I had no way to verify.

Reply:
@shushanhf please check if the main branch build is fixed after changing it to `PTFF_THREAD_HIJACK_HI`. In riscv64, I have just used `PTFF_THREAD_HIJACK_HI`, but haven't tested the nativeaot implementation due to other issues we have in that port at the moment.